### ISSUE TITLE
Update all MOF Class versions to 1.0.0.0

### ISSUE THREAD

**Issue by PlagueHO** [PowerShell/xStorage#57]
Currently all MOF Classes in xStorage are set to version 0.1.0.0.

I don't believe this should be the case for a released Resource. These should all be upgraded to 1.0.0.0

I'd propose doing this as part of #53.

**Comment by PlagueHO** [comment]
@TravisEz13, @kwirkykat  - any objections to doing this?

**Comment by kwirkykat** [comment]
@PlagueHO No objections as long as you're planning for the HQRM to release not under 'preview' (e.g. release as 1.0.0.0 instead of 0.1.0.0)

**Comment by PlagueHO** [comment]
Hi @kwirkykat  - yes, definitely. Once #53 is completed (I'm including many of the fixes and feature requests) then I think we should consider this a RTM version (with your teams consent of course). I should have complete unit test coverage and complete integration coverage (there are some limitations on int. tests though).

I won't rename the resource using HQRM naming though, I'll leave that as a separate PR that could be created after #53.

**Comment by kwirkykat** [comment]
:+1:
